libraries/plugins/apis/database_api/include/steem/plugins/database_api/database_api_args.hpp

```diff
@@ -556,6 +556,23 @@ struct verify_signatures_return
    bool valid;
 };
 
+/*struct find_transaction_args
+{
+   transaction_id_type trx_id;
+   uint32_t      expiration;
+};*/
+struct find_transaction_args
+{
+    fc::variant  trx_id;
+    fc::variant  expiration;
+};
+    
+struct find_transaction_return
+{
+   uint32_t trx_status_code;
```

Comment:
Return the enum and reflect the enum. In binary, it will be the int status, but will serialize as the enum types in JSON.

Reply:
done, returning enum.